### PR TITLE
Fixes #12019: Rudder 4.3 fails to build for multiple reasons

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/12019

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -118,6 +122,12 @@ Requires: perl
 BuildRequires: pam-devel
 %endif
 
+## Requirement for cpanminus
+# rh 5,6,7, sles10
```
Comment (amousset) [inline rudder-agent/SPECS/rudder-agent.spec]: -sles10

Comment (amousset) [reply rudder-agent/SPECS/rudder-agent.spec]: -rhel5

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**